docs/dg/dev/architecture/architectural-convention.md

```diff
@@ -2002,23 +2010,21 @@ The components below are essential in `core module development` for the purpose
                 └── [Module]To[Module]ServiceInterface.php  
 ```
 
-**Description**
+According to the Interface Segregation Principle, every module defines an interface for each external class it relies on. To decouple and encapsulate these dependencies effectively, `Bridges` are used. These `Bridges` serve to wrap dependencies such as [Facade](#facade-design-pattern), [Client](#facade-design-pattern), [Service](#facade-design-pattern), external library classes, and more. By doing so, a module explicitly declares its own requirements, enabling any class that implements the interface to be used. This approach facilitates seamless module decoupling, allowing both modules to evolve independently while maintaining a high degree of flexibility and maintainability.
 
-According to the Interface Segregation Principle, every module defines an interface for each external class it relies upon. To decouple and encapsulate these dependencies effectively, we utilize `Bridges`. These `Bridges` serve to wrap dependencies such as [Facade](#facade-design-pattern), [Client](#facade-design-pattern), [Service](#facade-design-pattern), external library classes, and more. By doing so, a module explicitly declares its own requirements, enabling any class that implements the interface to be utilized. This approach facilitates seamless module decoupling, allowing both modules to evolve independently while maintaining a high degree of flexibility and maintainability.
+#### Conventions
 
-**Conventions**
-
-- The `Bridge` class needs to define and implement an interface that holds the specification of each `public` method (mind the missing `bridge` suffix word in the interface name).
+- The `Bridge` class needs to define and implement an interface that holds the specification of each `public` method. Mind the missing `bridge` suffix word in the interface name.
 
-<details><summary markdown='span'>Additional Conventions for Module Development and Core Module Development</summary>
+<details><summary>For *module development* and *core module development*</summary>
 
-- `Bridges` must contain only the delegation logic to the friend method.<br/>
-- The `Bridge` constructor can not have parameters to avoid coupling to a specific class.<br/>
-- `Bridge` classes and interfaces need to be declared as strict as possible for input arguments and returns values, compared to their friend method with valid types (no `mixed`, no `x|y`).<br/>
+- `Bridges` must contain only the delegation logic to the friend method.
+- The `Bridge` constructor can't have parameters to avoid coupling to a specific class.
+- `Bridge` classes and interfaces need to be declared as strict as possible for input arguments and returns values, compared to their friend method with valid types (no `mixed`, no `x|y`).
```

Comment:
Break this down into several sentences

Reply:
- Bridge` classes and interfaces need to be declared as strict as possible with valid types over input arguments and return values based on their friend method (eg. no `mixed`, no `x|y`).